jsk_fetch_robot/jsk_fetch_startup/scripts/odometry_transformer.py

```diff
@@ -80,17 +80,17 @@ def __init__(self):
     def cb_odometry(self, msg):
 
         # Nan check
-        if math.isnan( msg.pose.pose.position.x ) or
-           math.isnan( msg.pose.pose.position.y ) or
-           math.isnan( msg.pose.pose.position.z ) or
-           math.isnan( msg.pose.pose.orientation.x ) or
-           math.isnan( msg.pose.pose.orientation.y ) or
-           math.isnan( msg.pose.pose.orientation.z ) or
-           math.isnan( msg.twist.twist.linear.x ) or
-           math.isnan( msg.twist.twist.linear.y ) or
-           math.isnan( msg.twist.twist.linear.z ) or
-           math.isnan( msg.twist.twist.angular.x ) or
-           math.isnan( msg.twist.twist.angular.y ) or
+        if math.isnan( msg.pose.pose.position.x ) or \
+           math.isnan( msg.pose.pose.position.y ) or \
+           math.isnan( msg.pose.pose.position.z ) or \
+           math.isnan( msg.pose.pose.orientation.x ) or \
+           math.isnan( msg.pose.pose.orientation.y ) or \
+           math.isnan( msg.pose.pose.orientation.z ) or \
+           math.isnan( msg.twist.twist.linear.x ) or \
+           math.isnan( msg.twist.twist.linear.y ) or \
+           math.isnan( msg.twist.twist.linear.z ) or \
+           math.isnan( msg.twist.twist.angular.x ) or \
+           math.isnan( msg.twist.twist.angular.y ) or \
            math.isnan( msg.twist.twist.angular.z ):
```

Comment:
@708yamaguchi how about this?

```suggestion
        if (math.isnan( msg.pose.pose.position.x ) or
            math.isnan( msg.pose.pose.position.y ) or
            math.isnan( msg.pose.pose.position.z ) or
            math.isnan( msg.pose.pose.orientation.x ) or
            math.isnan( msg.pose.pose.orientation.y ) or
            math.isnan( msg.pose.pose.orientation.z ) or
            math.isnan( msg.twist.twist.linear.x ) or
            math.isnan( msg.twist.twist.linear.y ) or
            math.isnan( msg.twist.twist.linear.z ) or
            math.isnan( msg.twist.twist.angular.x ) or
            math.isnan( msg.twist.twist.angular.y ) or
            math.isnan( msg.twist.twist.angular.z )):
```

Reply:
Thank you for your advice.
I updated the branch.